### PR TITLE
Change to using My- as a prefix for the case study protocol names

### PR DESCRIPTION
**Pull request by shabalind**
Previously we used either Custom- prefix, or -2 suffix or neither of the above.

This change migrates *all* protocol names to use My- prefix instead. It's quite short and makes it clear that case study protocols are all defined within the examples module and are different than analogous protocols in standard library or in S4TF.

### CODE REVIEW

File: Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift [hunk 1]
```diff
@@ -28,652 +28,651 @@ func specializedAdd(_ lhs: BinaryTree<Double>, _ rhs: BinaryTree<Double>) -> Bin
...
 func specializedAdd(_ lhs: Point2, _ rhs: Point2) -> Point2 {
     return Point2(
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
-        _1: lhs._1 + rhs._1
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 38)
-        ,
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
+        // ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
+        _1: lhs._1
+            + rhs._1// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 38)
```
Comment (dabrahams) [inline Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift]: Did the version of gyb change?  This looks like a bug; souceLocation comments are supposed to be on their own lines.

**Review by dabrahams — APPROVED** [review]
In principle LGTM, but I'm concerned about these changes to the generated `// ###sourceLocation` comments.  Frankly if we're going to check in gyb output, we should always pass `--line-directive ''` on the command line to just strip these things out.

**Comment by dabrahams** [comment]
Oh, it's the "reformat the code" commit that's the culprit.  Don't reformat generated code please.  It just creates diff churn in the repository.

**Comment by shabalind** [comment]
I just rerun the gyb with `--line-directive ''` and reformatted the result. It should reduce the diff churn going forward.

**Pull request merged**